modules/instrain/profile/main.nf

```diff
@@ -0,0 +1,46 @@
+process INSTRAIN_PROFILE {
+    tag "$meta.id"
+    label 'process_high'
+    label 'error_retry'
+    label 'process_long'
```

Comment:
I'm not sure if adding both the high and long labels will work here, do you know if the order of the multiple labels have an inherant priority order?

Reply:
(as in it's the first time I've seen this, so  I genuinely don't know!)

Reply:
I'm not sure, but I ended up just removing the 'error_retry' and 'process_long' labels; I really don't think they're needed anyways